### PR TITLE
refactor: streamline routes

### PR DESCRIPTION
**Pull request by arleytm**

### CODE REVIEW

File: index.js
Comment (hm23) [inline index.js:28]: No need to not merge this, but maybe we should clarify if @danjoa prefers `GraphQLAdapter` or `CDSGraphQLAdapter` here.

**Review by hm23 — APPROVED** [review]

**Comment by hm23** [comment]
Sorry, forgot to check this before hitting approve: I think we should mention this in the changelog. Marcel mentioned that some users are currently requiring the removed `lib.index`.

**Comment by arleytm** [comment]
> Sorry, forgot to check this before hitting approve: I think we should mention this in the changelog. Marcel mentioned that some users are currently requiring the removed `lib/index`.

Example usage of the deprecated module:
https://github.com/SAP-samples/hana-developer-cli-tool-example/blob/bff8500d67ae4e29ec575b939cf7576dbfdb5820/bin/cds.js#L394-L399

Please note that this module was originally intended for internal use only, and its public usage resulted in incomplete functionality.

**Pull request merged**
